components/badge/badge.css

```diff
@@ -0,0 +1,147 @@
+:root{
+    --blueviolet:blueviolet;
+    --pink:pink;
+    --orange:orange;
+    --red:red;
+    --white:white;
+    --green:rgb(21, 179, 21);
+    --blue:rgb(37, 171, 224) ;
+    --aqua:aqua;
+    --grey:grey;
+}
+
+.badge-container {
+    display: flex;
+    gap: 1rem;
+    align-items: baseline;
+    border:1px solid var(--grey);
+    padding:25px;
+    width:80%;
+    background-color: rgb(248, 244, 244);
+    margin-bottom:1rem;
+  }
+  
+  .badge-parent {
+    position: relative;
+    max-width: fit-content;
+    padding: 3px;
+  }
+  
+  .badge {
+    position: absolute;
+    background-color: var(--red);
+    color: var(--white);
+    max-width: fit-content;
+    border-radius: 50%;
+    text-align: center;
+    padding: 5px;
+    top: 0px;
+    right: 0px;
+    border: 2px solid var(--white);
+  }
+  
+  .bell-badge {
+    height: 30px;
+    width: 3rem;
+    left: 2rem;
```

Comment:
Either use px or rems, here you have used both

Reply:
i have fixed it